frontends/web/src/routes/account/receive/receive-bb01.tsx

```diff
@@ -249,12 +249,13 @@ export const Receive = ({
                   { forceVerification && verifying && (
                     <div className={style.hide}></div>
                   )}
-                  { account && forceVerification && verifying && (
-                    <Dialog
-                      title={verifyLabel}
-                      disableEscape={true}
-                      medium centered>
-                      <div className="text-center">
+                  <Dialog
+                    open={(!!(account && forceVerification && verifying))}
```

Comment:
Nit: as those are all booleans anyways it's fine to not use `!!`

Reply:
`account` isn't actually boolean, it's `accountApi.IAccount | undefined`..

We could do `(!!account) && forceVerification && verifying` though.